src/Sarif.Multitool.FunctionalTests/Rules/EndLineMustNotBeLessThanStartLineTests.cs

```diff
@@ -38,16 +38,10 @@ public void EndLineMustNotBeLessThanStartLine_EndLineNotSpecified()
             Verify(new EndLineMustNotBeLessThanStartLine(), "EndLineNotSpecified.sarif");
         }
 
-        [Fact(DisplayName = nameof(EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInAnalysisTarget))]
-        public void EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInAnalysisTarget()
+        [Fact(DisplayName = nameof(EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInResultLocation))]
+        public void EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInResultLocation()
         {
-            Verify(new EndLineMustNotBeLessThanStartLine(), "EndLineLessThanStartLineInAnalysisTarget.sarif");
-        }
-
-        [Fact(DisplayName = nameof(EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInResultFile))]
-        public void EndLineMustNotBeLessThanStartLine_EndLineLessThanStartLineInResultFile()
-        {
-            Verify(new EndLineMustNotBeLessThanStartLine(), "EndLineLessThanStartLineInResultFile.sarif");
+            Verify(new EndLineMustNotBeLessThanStartLine(), "EndLineLessThanStartLineInResultLocation.sarif");
```

Comment:
As in the previous PR: "`analysisTarget`" is now a `fileLocation`, so it has no `region`, so this rule does not apply to it, so remove that test. And "result file" no longer appears in the spec; the test applies to locations within a result, hence "ResultLocation".